src/JsVlcPlayer.cpp

```diff
@@ -204,6 +207,8 @@ void JsVlcPlayer::initJsApi( const v8::Handle<v8::Object>& exports )
 
     Local<Function> constructor = constructorTemplate->GetFunction();
     _jsConstructor.Reset( isolate, constructor );
+    SET_RO_PROPERTY( exports, "vlcVersion", &JsVlcPlayer::vlcVersion );
+    SET_RO_PROPERTY( exports, "vlcChangeset", &JsVlcPlayer::vlcChangeset );
```

Comment:
This is not correct, since there are no C++ class instance attached to `exports`. I had to add some sort of check for it in  `SET_RO_PROPERTY`...
And maybe even better choice will be just set `version` and `changeset` as string constants, since it could not be changed during one session... I'll think how to do it better.